engine/link/mystery_gift.asm

```diff
@@ -33,18 +33,26 @@ DoMysteryGift:
 	ld de, .String_PressAToLink_BToCancel
 	call PlaceString
 	call WaitBGMap
-
+	
```

Comment:
Remove any trailing whitespace.

Reply:
c829450c7431ce2f1f4a3e482630cb0a513c50b8 should resolve this.